src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/AttackSystem.java

```diff
@@ -55,42 +57,58 @@
     @In
     EntityManager entityManager;
     @In
-    private LocalPlayer localPlayer;
-    @In
     private WorldProvider worldProvider;
     @In
     private BlockManager blockManager;
 
     private EntityRef flagSlot;
     private EntityRef item;
-
+    private EntityBuilder builder;
+    private Iterable<EntityRef> particleEntities;
```

Comment:
Any reason why this is global when usage is local to the `removeParticleEmitterFromPlayer` method